source/controlTypes/processing.py

```diff
@@ -0,0 +1,216 @@
+# A part of NonVisual Desktop Access (NVDA)
```

Comment:
I think this file deserves a more descriptive name.

Reply:
Do you have a suggestion? I think it's possible we make `processPositiveStates` and `processNegativeStates` private (with underscores) and rename the file `processAndLabelStates.py`. These functions appear to just be helper functions for `processAndLabelStates`. Any API breakages will have to be delayed until 2022.1, so they can be renamed in `__init__.py`

Reply:
Yep, I'm happy with that idea.